inscriptions/ISic3011.xml

```diff
@@ -183,40 +188,22 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI329738">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Πυρ
-                    <supplied reason="lost">— — — — —</supplied>
-                    <lb n="2"/>ΑΓ
-                    <supplied reason="lost">— — — — — —</supplied>
-                    <lb n="3"/>
-                </ab>
-            </div>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI331243">
-                <ab>
-                    <lb n="1"/>
-
-
-                     σπέ<supplied reason="lost">νδεται(?)</supplied>·<lb n="2"/> Πῦρ·<lb n="3"/>Ἄρκο.<lb n="4"/>ἐξο
-                    <supplied reason="lost" cert="low">υσίαι</supplied>
-                    <lb n="5"/>
-                    <supplied reason="lost">— — κ</supplied>α
-                    <supplied reason="lost">ὶ</supplied>
-                    <lb n="6"/>
-                    <supplied reason="lost">Δε</supplied>
-                    <expan>
-                        <abbr>ιφέλοˉ</abbr>
-                        <ex>?</ex>
-                    </expan>
-                    .
-                    <lb n="7"/>
+                    <lb n="1"/>σπέ<supplied reason="lost">νδεται</supplied>
+                    <lb n="2"/>Πῦρ
+                    <lb n="3"/>Ἄρκο
+                    <lb n="4"/>ἐξο<supplied reason="lost" cert="low">υσί</supplied>α<supplied reason="lost" cert="low">ι</supplied>
+                    <lb n="5"/><gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost">κ</supplied>α<supplied reason="lost">ὶ</supplied>
+                    <lb n="6"/><supplied reason="lost">Δε</supplied>ιφέλο̄
```

Comment:
Hmmm. This looks like a very confused record to me. lines 1-3 are on one block on one side of the altar; lines 4-6 are on a different block on a different side of the altar, and there is no actual suggestion that these are part fo the same inscription. I think this record is going to need to be split up and then is a nice case study for implementing xml:id cross-referencing (@simonastoyanova) between the two files. This record however could remain as the record for lines 1-3, since that was what it originally contained, and we'll create a new record for lines 4-6. The Manganaro dsicussion is impressively unclear; I will make a note to get hold of the original Neutsch publication on my next library visit. I think it would be best to reduce this to lines 1-3, and comment out lines 4-6 with a note that this is a separate text to be put in a separate file - and I'll sort it out next week.